web/.env.example

```
@@ -0,0 +1,16 @@
+# --- Required Environment Variables for Vite ---
+
+# The base URL for the backend API
+VITE_API_URL="http://localhost:4000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)